Source/Rows/Common/FieldRow.swift

```diff
@@ -382,6 +396,27 @@ open class _FieldCell<T> : Cell<T>, UITextFieldDelegate, TextFieldCell where T:
 
     open func textFieldShouldEndEditing(_ textField: UITextField) -> Bool {
         return formViewController()?.textInputShouldEndEditing(textField, cell: self) ?? true
-    }
-
+	}
+	
+	open override func layoutSubviews() {
+		super.layoutSubviews()
+		guard let row = (row as? FieldRowConformance) else { return }
+		let titlePercentage = row.titlePercentage ?? 0.3
```

Comment:
As described [here](https://github.com/xmartlabs/Eureka/pull/965#issuecomment-318293995), I would go for this:

`let titlePercentage = row.titlePercentage ?? 0.7`